### ISSUE TITLE
improvement suggestion to tabs handling

### ISSUE THREAD

**Issue by danmarsden** [patrickthibaudeau/moodle-format_glendon#5]
After converting to use the glendon course format I was initially confused when I selected a section and no content was displayed. I had to go looking at the docs to find out that I have to create "tabs" using the extra plugin. I would suggest that if no tabs exist you should display all content from that section within the page (and not display tabs.) This would also have the result of not requiring your other plugin to use the glendon course format - you would only need the plugin to enable the "tabs" features within the sections.

**Comment by patrickthibaudeau** [comment]
You are correct, however, the whole idea was to use tabs and to avid displaying the whole section in a linear way. The other reason was to easily hide resources and activities without having to use the built-in feature. The professors using this format find it much faster to simply drag the activities they want hidden before the first tab.

Having said this, I still like the suggestion and will find a way to implement it.
